### PR TITLE
docker: there is now the official repo with images

### PR DESCRIPTION
**Pull request by abitrolly**
https://github.com/canonical/snapcraft-rocks/

- [x] Have you followed the [guidelines for contributing](https://github.com/snapcore/snapcraft/blob/master/CONTRIBUTING.md)?
- [x] Have you signed the [CLA](http://www.ubuntu.com/legal/contributors/)?
- [x] Have you successfully run `tox run -m lint`?
- [x] Have you successfully run `tox run -e test-py310`? (supported versions: `py39`, `py310`, `py311`, `py312`)

-----

### CODE REVIEW

File: docker/README.md [hunk 1]
```diff
@@ -1,16 +1,26 @@
-# Creating docker containers for snapcraft
+# Docker images for `snapcraft`
 
-By default the `Dockerfile` builds Ubuntu 16.04 (Xenial) image with `snapcraft` from the `edge` channel.
+OCI-compliant container images and their sources are officially supported by
```
Comment (lengau) [inline docker/README.md:3]: @tigarmo how "official" are the snapcraft-rocks?

Comment (abitrolly) [reply docker/README.md:3]: @lengau it is being the repo of verified organization https://github.com/canonical. Is that not enough?

Or maybe you're also confused by  "maintained by the Starcraft team"? )

image

The screenshot of what Google and Blizzard StartCraft team were doing for AI in 2016. )

Comment (tigarmo) [inline docker/README.md:3]:
```suggestion
OCI-compliant container images and their sources are officially supported by the
```

Comment (tigarmo) [reply docker/README.md:3]: > @tigarmo how "official" are the snapcraft-rocks?

Well it's the way the Starcraft team decided to support Snapcraft images, so as official as it gets I guess. I've been slacking on supporting the rocks but that's a separate issue.

**Review by tigarmo — APPROVED** [review]
thanks for this!

**Comment by abitrolly** [comment]
@tigarmo no problem. I am itching my own itches. )

**Review by lengau — APPROVED** [review]

**Review by mr-cal — APPROVED** [review]
Thanks!

**Pull request merged**